test/SlowTests/Issues/RavenDB_3647.cs

```diff
@@ -1,104 +0,0 @@
-using System.Collections.Generic;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Operations.Indexes;
-using Raven.Client.Documents.Operations.Transformers;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDB_3647 : RavenTestBase
-    {
-        [Fact]
-        public void CanLockTransformers()
-        {
-            using (var store = GetDocumentStore())
-            {
-                store.ExecuteTransformer(new SimpleTransformer());
-                //Checking that we can lock transformer
-                store.Admin.Send(new SetTransformerLockOperation("SimpleTransformer", TransformerLockMode.LockedIgnore));
-                var transformerDefinition = store.Admin.Send(new GetTransformerOperation("SimpleTransformer"));
-                var oldTransformResults = transformerDefinition.TransformResults;
-                Assert.Equal(transformerDefinition.LockMode, TransformerLockMode.LockedIgnore);
-                //Checking that we can't change a locked transformer
-                transformerDefinition.TransformResults = NewTransformResults;
-                store.Admin.Send(new PutTransformerOperation(transformerDefinition));
-                transformerDefinition = store.Admin.Send(new GetTransformerOperation("SimpleTransformer"));
-                Assert.Equal(transformerDefinition.TransformResults, oldTransformResults);
-                //Checking that we can unlock a transformer
-                store.Admin.Send(new SetTransformerLockOperation("SimpleTransformer", TransformerLockMode.Unlock));
-                transformerDefinition = store.Admin.Send(new GetTransformerOperation("SimpleTransformer"));
-                Assert.Equal(transformerDefinition.LockMode, TransformerLockMode.Unlock);
-                //checking that the transformer is indeed overridden
-                transformerDefinition.TransformResults = NewTransformResults;
-                store.Admin.Send(new PutTransformerOperation(transformerDefinition));
-                transformerDefinition = store.Admin.Send(new GetTransformerOperation("SimpleTransformer"));
-                Assert.Equal(NewTransformResults, transformerDefinition.TransformResults);
-            }
-        }
-
-        [Fact]
-        public void CanLockIndexes()
```

Comment:
I don't think this test should be removed - it operates on indexes, not transformers